modules/reference/pages/cluster-properties.adoc

```diff
@@ -1002,7 +1002,7 @@ The topic property xref:./topic-properties.adoc#segmentms[`segment.ms`] override
 
 *Default*: null
 
-*Range*: [60000 (60 sec), ...]
+*Range*: [600000 (10 minutes), ...]
```

Comment:
https://github.com/redpanda-data/redpanda/blob/0afd0bd1ab7a012149b460639fe525d82299992f/src/v/config/configuration.cc#L89-L98

The default value is 2 weeks (in milliseconds: 1209600000), and the range didn't change: any value under 60 seconds (in milliseconds 60000) is rejected.

the values of this prop haven't changed since the first release it landed on (23.1 if I remember correctly, but I might be wrong)

```suggestion
*Default*: 1209600000 (2 weeks)

*Range*: [60000 (60 sec), ...]
```